<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-UCE2Q720" ignored="0" name="Test" tests="3" failures="1" timestamp="2021-04-19T18:42:34 CEST" time="0.041" errors="0">
  <testcase name="failureTest" time="0.008" classname="practica10.ejemplo5_ISuiterListener.SalesforceTest">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at practica10.ejemplo5_ISuiterListener.SalesforceTest.failureTest(SalesforceTest.java:22)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- failureTest -->
  <testcase name="skippedTest" time="0.001" classname="practica10.ejemplo5_ISuiterListener.SalesforceTest">
    <skipped/>
  </testcase> <!-- skippedTest -->
  <testcase name="successTest" time="0.001" classname="practica10.ejemplo5_ISuiterListener.SalesforceTest"/>
</testsuite> <!-- Test -->
